src/acquisition/covid_hosp/common/utils.py

```diff
@@ -168,7 +163,7 @@ def merge_by_key_cols(dfs, key_cols, logger=False):
     return result.reset_index(level=key_cols)
 
   @staticmethod
-  def update_dataset(database, network, newer_than=None, older_than=None):
+  def update_dataset(database, network=Network, newer_than=None, older_than=None):
```

Comment:
is this `network` argument here just to do dependency injection in tests?  can we remove it from the function prototype and use `unittest.mock.patch` instead?